include/SSVOpenHexagon/Core/HexagonGame.hpp

```diff
@@ -160,13 +160,46 @@ class HexagonGame
     template <typename T, typename... TArgs>
     T runLuaFunction(const std::string& mName, const TArgs&... mArgs)
     {
-        return Utils::runLuaFunction<T, TArgs...>(lua, mName, mArgs...);
+        try {
+            return Utils::runLuaFunction<T, TArgs...>(lua, mName, mArgs...);
+        } 
+        catch(std::runtime_error& mError)
+        {
+            std::cout << "[Lua] Runtime error on \""
+                      << mName
+                      << "\" with level \""
+                      << levelData -> name
+                      << "\": \n"
+                      << ssvu::toStr(mError.what()) << "\n"
```

Comment:
Do we need the `toStr` here?

Reply:
I'm not sure. This was in the legacy implementation of reporting Lua errors, and it looks like what() returns a pointer to an array of characters, so if you want to consider this a string, feel free to make the changes.